README.md

```diff
@@ -244,6 +244,40 @@ type Output struct {
 }
 ```
 
+### Struct field name folding
+
+With `goverter:fold` tag you can instruct goverter to perform case-insensitive mapping between
```

Comment:
I find `fold` not that expressive, because the meaning in this context is not 100% clear at least to me. How about `goverter:matchIgnoreCase`, `goverter:match caseInsensitive`, `goverter:matchLenient`, `goverter:match ignoreCase` or `goverter:match lenient`?

Reply:
I changed to goverter:matchIgnoreCase.